Orbital-Docs/_config.yml

```diff
@@ -18,30 +18,30 @@
 # You can create any custom variable you would like, and they will be accessible
 # in the templates via {{ site.myvariable }}.
 
-title: Orbital
 email: opensource@focisolutions.com
 description: # this means to ignore newlines until "baseurl:"
-baseurl: '' # the subpath of your site, e.g. /blog
-url: '' # the base hostname & protocol for your site, e.g. http://example.com
-twitter_username: jekyllrb
+baseurl: "" # the subpath of your site, e.g. /blog
+url: "" # the base hostname & protocol for your site, e.g. http://example.com
+title: ""
 github_username: FociSolutions
 minimal_mistakes_skin: 'contrast'
 search: true
+logo: /assets/images/logo.png
 footer:
   links:
-    - label: 'GitHub'
+    - label: "GitHub"
       icon: fab fa-fw fa-github
-      url: 'https://github.com/FociSolutions/Orbital'
-    - label: 'Contribute'
-      url: '/docs/contribute'
-    - label: 'Contact Us'
-      url: 'https://github.com/FociSolutions/Orbital'
+      url: "https://github.com/FociSolutions/Orbital"
+    - label: "Contribute"
```

Comment:
missing to return the single quotes in the rest of the document

Reply:
Fixed